src/platforms/kotlin/config.yml

```diff
@@ -0,0 +1,10 @@
+title: Kotlin
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.kotlin
+fallbackPlatform: java
+categories:
+  - mobile
+keywords:
+  - kotlin
+  - ndk
```

Comment:
this is more Android related

Reply:
Agreed. I'll remove NDK. Kotlin currently doesn't have search results, so that will stay.